src/app/board/page.tsx

```diff
@@ -0,0 +1,288 @@
+'use client';
+
+// react, next
+import { useState, useRef, useCallback, FormEvent, useEffect } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+
+// 외부 라이브러리
+import { format } from 'date-fns';
+import { throttle } from 'lodash';
+
+// 로컬 파일
+import { getArticles } from '@/api';
+import useAsync from '@/hooks/useAsync';
+import { Article } from '@/types/article';
+
+// 컴포넌트
+import Dropdown from '@/components/Dropdown';
+
+const PAGESIZE = 10;
+
+// 디바이스 크기 별로 베스트 게시글 개수 설정
+const getBestArticles = (): number => {
+  const width = window.innerWidth;
+  if (width >= 1280) {
+    return 3;
+  } else if (width >= 768) {
+    return 2;
+  } else {
+    return 1;
+  }
+};
+
+export default function Page() {
+  const [resultArticle, setResultArticle] = useState<Article[] | []>([]);
```

Comment:
감사합니다!

커멘트로 남겨주신 부분 여기에 답글 남깁니다!

가끔 발생한다는 중복 key 는 어떤 상황에 발생하는지 알 수 있을까요?
지금 코드상으로는 문제 없어보이고, 배포된 url 은 접근 권한이 없어서 확인해볼수가 없네요 ㅠㅠ

>>
Encountered two children with the same key, `223`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

이라는 에러가 뜨네요 ㅠ
아마 초기 렌더링 중에 새로고침을 하거나 최신순/좋아요 순으로 리렌더링 시 저렇게 뜨는 거 같은데 방법이 있을까요?
api 요청을 다 불러오고 아이템이 다 로드되었을 때 화면이 떴으면 좋겠는데, 오늘 멘토링 시간에 배운 loading.tsx를 만들어 보는 게 좋을까요?